app/controllers/cards_controller.rb

```diff
@@ -0,0 +1,14 @@
+class CardsController < ApplicationController
+
+  def index
+    @cards = Card.all
+  end
+
+  def create
+    a = Card.new
+    a['original_text']        = params['original_text']
+    a['translated_text'] = params['translated_text']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.